app/controllers/claims/support/schools/mentors_controller.rb

```diff
@@ -8,6 +8,22 @@ def index
     @pagy, @mentors = pagy(@school.mentors.order(:first_name, :last_name))
   end
 
+  def new
+    mentor_form
```

Comment:
This is a personal opinion, not really a blocker

Can we declare a ` helper_method :mentor_form` at the top of this controller and have the new method empty? `def new; end` It will be consistent with the claim controller